centrapay/centrapay-docs
# Make navigation dynamically pick up pages in the content folder

We cut some corners by making the navigation static to hit some deadlines.

Having a static navigation menu will mean that when adding pages, there will need to be additional steps followed to add the page to the left hand navigation menu on the site.

Making the navigation menu dynamic means that when adding pages, you will only have to define where it should sit in the nav. The new page will automatically appear on the navigation menu on the site.

Adding new sections will still need to be manual. This is a fine compromise because new sections are rare.

Test plan:
- Ensure that there are no regressions in the site
- Add a new page with a navPath --> this page should automatically appear in the nav menu on the site
